src/constants/task.ts

```diff
@@ -6,7 +6,7 @@ const { gray, green, purple, red, yellow } = palette;
 
 export const taskStatusToCopy = {
   [TaskStatus.ScheduledUmbrella]: "Scheduled",
-  [TaskStatus.FailedUmbrella]: "Failed / Timed Out",
+  [TaskStatus.FailedUmbrella]: "Failed",
```

Comment:
The new font is wider so it overflowed the boxes :( since other status badges are umbrellas that represent more than just the title I figured it was a change with precent.